Game/Source/GDKTestGymsEditor.Target.cs

```diff
@@ -10,6 +10,7 @@ public GDKTestGymsEditorTarget(TargetInfo Target) : base(Target)
 		Type = TargetType.Editor;
 		ExtraModuleNames.Add("GDKTestGyms");
 		ExtraModuleNames.Add("GDKTestGymsFunctionalTests");
+
 		DefaultBuildSettings = BuildSettingsVersion.V2;
```

Comment:
Hm, I see I conflicted with changes from master here, so someone had the same idea.
Not sure if it's necessary for all targets?

Reply:
I think you're right, I should have set it in all targets